### PR TITLE
As4242421022 dal1 - change from IPv6 LL to ULA

### PR DESCRIPTION
**Pull request by bsdrocker**

### CODE REVIEW

File: routers/router.dal1.yml [hunk 1]
```diff
@@ -1,4 +1,16 @@
 ---
+- name: BSDROCKER
+  asn: 4242421022
+  ipv6: fd2e:5e5e:83ad::1022:207
+  multiprotocol: true
+  extended_nexthop: true
```
Comment (jvoss) [inline routers/router.dal1.yml:6]:
```suggestion
  ipv4: <FIXME: add an IPv4 address for this tunnel>
  ipv6: fd2e:5e5e:83ad::1022:207
  multiprotocol: true
```

Let's try this without enabling `extended_nexthop` given the difficulties in #216.

**Review by jvoss — CHANGES_REQUESTED** [review]

**Review by jvoss — APPROVED** [review]

**Comment by jvoss** [comment]
@bsdrocker this has been deployed.

With this configuration, remember that you will need a static route to our end of the wireguard tunnel:

```
ip -6 route fdb1:e72a:343d::5/128 via <your wireguard interface>
```

And then configure BGP peering to `fdb1:e72a:343d::5`. Note that you may need to enable multihop or disable connected checks depending on your situation.

Otherwise if you want to use a subnet on the tunnel such that each end is part of the same network we will need an address for our side and the subnet size.

**Comment by bsdrocker** [comment]
@jvoss I can't ping you and I also see BGP trying to establish from your side from the link-local address still. BGP on my side is stuck in Connect.

**Comment by jvoss** [comment]
Did you add the static route I mentioned?

Also the old peering is still configured and will be reconciled tonight after automation runs again.

**Comment by bsdrocker** [comment]
Indeed I have:

```
bsdrocker@dal-us-01:~$ ip -6 route |grep wg1
fd2e:5e5e:8a3d::1022:207 dev wg1 proto kernel metric 256 pref medium
fdb1:e72a:343d::5 dev wg1 proto kernel metric 256 pref medium
bsdrocker@dal-us-01:~$
```

```
bsdrocker@dal-us-01:~$ ping fdb1:e72a:343d::5 -I wg1
PING fdb1:e72a:343d::5 (fdb1:e72a:343d::5) from fd2e:5e5e:8a3d::1022:207 wg1: 56 data bytes
^C
--- fdb1:e72a:343d::5 ping statistics ---
5 packets transmitted, 0 received, 100% packet loss, time 4109ms

bsdrocker@dal-us-01:~$
```

**Comment by bsdrocker** [comment]
I realized what the issue is - there's actually a typo in the PR. I'll have to fix it.

**Pull request merged**